### ISSUE TITLE
Voice option? 

### ISSUE THREAD

**Issue by wcyun1** [tartanguru/alexa-assistant#72]
Great job and loving it so far, thanks so much Ross for the gift (:
Quick question though, can we change it so that the API call uses a male voice instead? Thanks!

**Comment by wcyun1** [comment]
Anyone? :)

**Comment by tartanguru** [comment]
The API doesn’t support this I’m afraid. The only option is to change the
language via the Google Assistant app

**Comment by wcyun1** [comment]
Yeah I've selected a male voice which took effect on both my phone and my Google Home Mini, but not on the Amazon Echo Dot when I tell Alexa to invoke the Google Assistant skill.. bummer!
